components/Hero.vue

```diff
@@ -32,13 +32,13 @@
     </div> -->
     <p class="hero-mailing-list-text">
       <span class="hero-mailing-list-text bold">
-        Registration is now live!</span><br>
+        Registration has closed!</span><br>
       Learn, Build, and Share with us at HackCamp 2020.
     </p>
     <div class="register-btn-container">
-      <button @click="register">
+      <!-- <button @click="register">
         REGISTER HERE
-      </button>
+      </button> -->
```

Comment:
Are we going to put the mailing list back instead or no?

Reply:
let's just remove the button for now bc it's a bit urgent to do so